index.html

```diff
@@ -1,9 +1,15 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>YOUR NAME HERE</title>
+  <title>Justin is learning</title>
 </head>
 <body>
-
+  <p>
+    TurnStone
+  </p>
+  <p>
+    ahza64
+  </p>
+  <img src="<iframe src="//giphy.com/embed/l0D7kU3kd1eDVLHxu" width="480" height="266" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="http://giphy.com/gifs/barack-obama-president-number-one-l0D7kU3kd1eDVLHxu">via GIPHY</a></p>">
```

Comment:
Hmm... Looks like you have an `iframe` embedded inside a string here. Did you check that you code works by opening this in your browser?